### ISSUE TITLE
Feature request: more languages at the time

### ISSUE THREAD

**Issue by Bazixv13** [tirkarthi/NotificationDictionary#50]
It would be nice if the user could select 2 or more languages for the dictionary at the same time and if the word is the same in both languages u get to choose a language in the notification via notification buttons

**Comment by j-lakeman** [comment]
I agree and would also very much appreciate this feature in the otherwise great app!
Any updates on this?

**Comment by tirkarthi** [comment]
Thanks for the feedback. Currently the language choice is stored a shared preference and notification bar has some limitation where 3 actions can be shown at max in some phones like Samsung Galaxy A54 I am using. There is already copy, share and read from notifications. Another part is that with shared preferences the db instance connection is selected and loaded. Each language is a separate db instance of its own so selecting a word and collating meanings might be difficult across databases since I am not aware of joins across databases unlike tables to implement this. I will leave an update here if I come across other approach here.

**Comment by j-lakeman** [comment]
Thanks for your answer!
How about querying two language databases concurrently? Returning two notifications if two successful hits occur.
